common/src/scheduler/task.rs

```diff
@@ -107,7 +108,8 @@ impl<AccountId> ScheduledTask<AccountId> {
             RemoveGasReservation(program_id, reservation_id) => {
                 handler.remove_gas_reservation(program_id, reservation_id)
             }
-            RemoveResumeSession(session_id) => handler.remove_resume_session(session_id),
+            #[allow(deprecated)]
+            RemoveResumeSession(_) => 0,
```

Comment:
do it on implementor side please as it's done for Pause program: log::debug + return zero